docs/notes/extensions.md

```diff
@@ -28,6 +28,10 @@ In addition, many plugins can be found on the [npm registry](https://www.npmjs.c
 
 ## Integrations
 
+## Omi (v5+)
```

Comment:
It needs an additional level `###` to be under the `Integrations` section. Can you also move this new section at the end, after `### Ember.js`.

Reply:
@dntzhang just push these changes to your `master` branch, you don't need to create a new PR :)

Reply:
ok. done~~